aten/src/ATen/native/mps/OperationUtils.mm

```diff
@@ -42,6 +43,13 @@ size_t compute_storage_numel_distance(const at::Tensor& t) {
   return rc;
 }
 
+/**
+ * Set contiguous strides to given tensor in-place.
+ */
+void restride_contiguous_(const at::Tensor& t) {
```

Comment:
This name is a bit confusing to me. When I saw it below, I though you actually shuffled the data around to restride the Tensor. But you just re-view the memory in a different way here.

Reply:
Yes I found it confusing in the first place but haven't come up with a more suitable name...